src/subtitles-octopus.js

```diff
@@ -253,9 +259,9 @@ var SubtitlesOctopus = function (options) {
         }
         if (self.debug) {
             var drawTime = Math.round(performance.now() - beforeDrawTime);
-            var blendTime = data.blendTime || 0;
-            if (blendTime > 0) {
-                console.log('render: ' + Math.round(data.spentTime - blendTime) + ' ms, blend: ' + Math.round(blendTime) + ' ms, draw: ' + drawTime + ' ms');
+            var blendTime = data.blendTime;
```

Comment:
Is this an unrelated change?

Reply:
Yes, a small one to slightly adjust the debug output :)